File: KPP_lab_1_2/KPP_lab_1_2/WorkerManager.cs
namespace KPP_lab_1_2
{
    internal class WorkerManager
    {
        public static List<Worker> generateWorkers()
        {
            List<Worker> workers = new List<Worker>();

            Worker ivan = new Worker("Ivan", "Ivanov", 1000, "LK", "523756", "Management University");
            ivan.Careers.Add(new Career(new DateTime(2010, 1, 1), new DateTime(2011, 1, 1), "Manager", "IT"));
            ivan.Careers.Add(new Career(new DateTime(2011, 1, 1), new DateTime(2012, 1, 1), "Senior Manager", "IT"));
            ivan.Characteristics.Add(new Characteristic("Hardworking", 5));
            ivan.Characteristics.Add(new Characteristic("Honest", 4));
            workers.Add(ivan);

            Worker petr = new Worker("Petr", "Petrov", 2000, "MP", "923856", "IT University");
            petr.Careers.Add(new Career(new DateTime(2013, 1, 1), new DateTime(2014, 1, 1), "Middle Developer", "IT"));
            petr.Careers.Add(new Career(new DateTime(2014, 1, 1), new DateTime(2015, 1, 1), "Senior Developer", "IT"));
            petr.Characteristics.Add(new Characteristic("Sociable", 4));
            petr.Characteristics.Add(new Characteristic("Stressfull", 3));
            workers.Add(petr);

            Worker jamal = new Worker("Jamal", "Jamalov", 3000, "AB", "323556", "Malahit University");
            jamal.Careers.Add(new Career(new DateTime(2015, 1, 1), new DateTime(2016, 1, 1), "Middle Developer", "IT"));
            jamal.Careers.Add(new Career(new DateTime(2016, 1, 1), new DateTime(2017, 1, 1), "Senior Developer", "IT"));
            jamal.Characteristics.Add(new Characteristic("Stressfull", 5));
            jamal.Characteristics.Add(new Characteristic("Honest", 5));
            workers.Add(jamal);

            return workers;
        }
    }
}

File: KPP1_5/src/main/java/com/university/util/WorkerManager.java
package com.university.util;

import com.university.entities.*;

import java.util.ArrayList;
import java.util.List;

public class WorkerManager {
    private WorkerManager() {
    }

    public static List<Worker> createWorkers() {
        List<Worker> workers = new ArrayList<>();

        List<Career> careerIvan = new ArrayList<>();
        careerIvan.add(new Career("2010-01-01", "2012-01-01", "Manager", "IT"));
        careerIvan.add(new Career("2012-01-01", "2014-01-01", "Senior Manager", "IT"));
        List<Characteristic> characteristicsIvan = new ArrayList<>();
        characteristicsIvan.add(new Characteristic("Honest", 5));
        characteristicsIvan.add(new Characteristic("Hardworking", 4));
        characteristicsIvan.add(new Characteristic("Responsible", 5));
        Worker workerIvan = new Worker("Ivan", "Ivanov", new PassportInfo("ZL", 954321),
                1000, "University Of IT", careerIvan, characteristicsIvan);
        workers.add(workerIvan);

        List<Career> careerPetr = new ArrayList<>();
        careerPetr.add(new Career("2014-01-01", "2016-02-05", "Human Resource Manager", "Management"));
        careerPetr.add(new Career("2016-02-05", "2018-01-01", "Senior Human Resource Manager", "Management"));
        List<Characteristic> characteristicsPetr = new ArrayList<>();
        characteristicsPetr.add(new Characteristic("Honest", 3));
        characteristicsPetr.add(new Characteristic("Stressful", 4));
        Worker workerPetr = new Worker("Petr", "Petrov", new PassportInfo("TM", 654321),
                2000, "University Of Management", careerPetr, characteristicsPetr);
        workers.add(workerPetr);

        List<Career> careerSidor = new ArrayList<>();
        careerSidor.add(new Career("2018-01-01", "2020-06-01", "Junior Developer", "IT"));
        List<Characteristic> characteristicsSidor = new ArrayList<>();
        characteristicsSidor.add(new Characteristic("Stressful", 4));
        characteristicsSidor.add(new Characteristic("Hardworking", 5));
        Worker workerSidor = new Worker("Sidor", "Apteka", new PassportInfo("CM", 987654),
                3000, "No University", careerSidor, characteristicsSidor);
        workers.add(workerSidor);

        List<Career> careerVasiliy = new ArrayList<>();
        careerVasiliy.add(new Career("2018-05-01", "2020-01-01", "Middle Developer", "IT"));
        careerVasiliy.add(new Career("2020-01-01", "2022-01-01", "Senior Developer", "IT"));
        List<Characteristic> characteristicsVasiliy = new ArrayList<>();
        characteristicsVasiliy.add(new Characteristic("Wild", 5));
        characteristicsVasiliy.add(new Characteristic("Sociability", 3));
        Worker workerVasiliy = new Worker("Vasiliy", "Apnequ", new PassportInfo("RM", 12854699),
                4000, "University Of Economics", careerVasiliy, characteristicsVasiliy);
        workers.add(workerVasiliy);
        return workers;
    }
}
